mystery_o_matic/output/latex/__init__.py

```
@@ -0,0 +1,148 @@
+        c = create_template(c).substitute(names_txt)
+        p = create_template(p).substitute(names_txt)
+        final_locations_map[c] = p
+
+    representations_map = {}
+    for l, r in locations.representations.items():
+        l = create_template("$" + l).substitute(names_txt)
+        representations_map[l] = r
+
+    # the weapons map will be computed reversing the location_weapons one
+    weapons_map = {}
+    for l, w in locations.weapon_locations.items():
```

Ruff

Ambiguous variable name: `l`

(E741)